### PR TITLE
Disable failing integration test for PG driver

### PR DESCRIPTION
**Pull request by sergiyvamz**
### Summary

Disable failing integration test for PG driver

### Additional Reviewers

@karenc-bq 
@aaron-congo 
@crystall-bitquill 
@brunos-bq 

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by karenc-bq — APPROVED** [review]

File: wrapper/src/test/java/integration/container/TestDriverProvider.java [hunk 1]
```diff
@@ -165,6 +165,10 @@ public void beforeEach(ExtensionContext context) throws Exception {
                       try {
                         instanceIDs = auroraUtil.getAuroraInstanceIds();
                       } catch (SQLException ex) {
+                        if ("28P01".equals(ex.getSQLState())) {
```
Comment (karenc-bq) [inline wrapper/src/test/java/integration/container/TestDriverProvider.java:168]: Could this error be thrown by RDS databases during switchover?

Comment (brunos-bq) [inline wrapper/src/test/java/integration/container/TestDriverProvider.java:168]: NIT: can we do a global static variable with the name of the error code? like POSTGRES_AUTH_ERROR_CODE

**Review by brunos-bq — APPROVED** [review]

File: wrapper/src/test/java/integration/container/TestDriverProvider.java (hunk 1 above)
Comment (sergiyvamz) [reply wrapper/src/test/java/integration/container/TestDriverProvider.java:168]: It's hard to say. The goal here is to detect authentication errors (i.e. test environment errors that are rare but may happen) and stop the tests.

Comment (sergiyvamz) [reply wrapper/src/test/java/integration/container/TestDriverProvider.java]: We know that error 28000 (not 28P01) is raised when TAZ cluster is switching over.

**Pull request merged**